src/pipe_mgr/shared/dal/dpdk/dal_init.c

```diff
@@ -117,6 +117,15 @@ int dal_enable_pipeline(bf_dev_id_t dev_id,
 		LOG_TRACE("Running command: %s\n", buffer);
 		status = system(buffer);
 		if (status) {
+			LOG_ERROR("%s line:%d  Command (%s) failed with exit code: %d\n",
+				  __func__, __LINE__, buffer, status);
```

Comment:
If I understand the system(3) man page correctly, it also sets the value of `errno` that might give additional details about the reason for these failures.  I would recommend printing `errno` here as well.

Reply:
Same suggestion for the `system()` call that is a few lines down from this comment.

Reply:
I have added it to them. It reports `errno=10 ECHILD`. I have no idea why it happens.

Reply:
I did a little bit of Google and ChatGPT4 searching on that question, and did not find a satisfying answer in about 10 minutes of trying.  One thing I often found was people suggestion NOT to use the `system()` API call at all, but other techniques of creating and waiting for child processes to finish.